misc/assets/inforequest-flow.svg

```diff
@@ -179,7 +179,8 @@
         <xhtml:div class="caption">Full Disclosure</xhtml:div>
       </xhtml:div>
     </foreignObject>
-    <path class="arrow" d="M -100,0 h -50 v -50 h 150 v 20" />
+    <path class="arrow" d="M -100,0 h -50 v 30 l 20,20" />
+    <path class="arrow" d="M 0,50 v -20" />
```

Comment:
Pls tu dolnu sipku tiez zalom, nech je to symetricke:

![flow](https://user-images.githubusercontent.com/5547275/71516581-129c3580-28aa-11ea-91b0-f700d35123ad.png)